lib/oli_web/live/products/payments/discounts/form.ex

```diff
@@ -40,7 +54,9 @@ defmodule OliWeb.Products.Payments.DiscountsForm do
         <button class="form-button btn btn-md btn-primary btn-block mt-3" type="submit">Save</button>
       </Form>
 
-      <button class="btn btn-md btn-outline-danger float-right mt-3" phx-click="clear" disabled={is_nil(@discount)}>Clear</button>
+      {#if @live_action == :institution}
```

Comment:
I think I prefer not to make buttons appear in a view when some dynamic condition inside the view changes (in this case to have discount or not), and always have it in the dom enabled/disabled. 

But 100% personal opinion, probably it's clearer to not show it (not very UI wise here 😄 )

Reply:
Had this same concern in the product image PR. I'm not fan of showing disabled buttons, but they're everywhere in the app today. Decided on keeping it for consistency sake. Agree that is very preference-based

Reply:
No strong opinion here and as @julmontesdeoca said this pattern is replicated in different places throughout the app. I just was trying out this feature locally and thought it'd be less confusing to hide the button in that case, but I'm ok with leaving this as is 🙂

Reply:
The reason we show disabled button is to give the user the indication an action exists but just may not be available in the current mode.

For example, I actually found myself struggling to find this discounts feature at all as an admin until I looked through the code and realized that I must first turn on "Requires Payment" for the product to even see the discounts page. I think it could be valid that an admin may want to manage or keep around discounts (or just understand that this feature exists) even when they have turned off "Requires Payment" for the section.

With that said, I am in favor of hiding buttons for different user types to keep the interface simple. For example, because this is an admin only feature, I think it makes sense to  only show it for admins, as it is not useful or other types of users (since they cannot access or use the feature)